ocaml/parsing/parsetree.mli

```diff
@@ -319,6 +319,10 @@ and expression_desc =
             - [fun ~l:P -> E1]
                       when [lbl] is {{!Asttypes.arg_label.Labelled}[Labelled l]}
                        and [exp0] is [None]
+            - [fun ~l:(P0 : [%src_pos]) -> E1]
```

Comment:
This isn't really a special case here. I would remove this comment, but keep the one below about what happens during type checking.